extmod/uasyncio/core.py

```diff
@@ -228,6 +239,24 @@ def run_until_complete(self, aw):
     def close(self):
         pass
 
+    def set_exception_handler(self, handler):
+        global _exc_handler
+        _exc_handler = handler
+
+    def get_exception_handler(self):
+        return _exc_handler if _exc_handler is not None else self.default_exception_handler
+
+    def default_exception_handler(self, context):
+        print(context["message"])
+        print("future:", context["future"], "coro=", context["future"].coro)
+        sys.print_exception(context["exception"])
+
+    def call_exception_handler(self, context):
+        if _exc_handler is None:
+            self.default_exception_handler(context)
+        else:
+            _exc_handler(self, context)
```

Comment:
This can be simplified to just the last line if `_ext_handler` has `default_exception_handler` set as the default/init value.